kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecutedCommandExec.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.operation
+
+import org.apache.hive.service.rpc.thrift.{TRowSet, TTableSchema}
+
+import org.apache.kyuubi.operation.FetchOrientation.FetchOrientation
+import org.apache.kyuubi.operation.log.OperationLog
+import org.apache.kyuubi.parser.plan.command.RunnableCommand
+import org.apache.kyuubi.parser.schema.SchemaHelper
+import org.apache.kyuubi.session.KyuubiSessionImpl
+
+class ExecutedCommandExec(
+    session: KyuubiSessionImpl,
+    override val shouldRunAsync: Boolean,
```

Comment:
do we need this, meta-operations are all false

Reply:
Asynchrony may be required given some of the new operations that will be supported in the future?